### PR TITLE
TunerStudio reconnect speedup

### PR DESCRIPTION
**Pull request by dron0gus**
-reduce timeouts ~~on both sides (may cause disconnect while flashing settings on boards with single bank flash)~~
-do not emit error messages and replies if connection is out of sync
-remove duplicated debug messages

### CODE REVIEW

File: firmware/tunerstudio/rusefi.input [hunk 1]
```diff
@@ -73,7 +73,7 @@ enable2ndByteCanID = false
 ;e.g. put writeblocks off and add an interwrite delay
    writeBlocks = on
    interWriteDelay = 10
-   blockReadTimeout    = 3000; Milliseconds general timeout
+   blockReadTimeout    = 300; Milliseconds general timeout
```
Comment (rusefillc) [inline firmware/tunerstudio/rusefi.input:76]: please fork this change into separate PR since this is questionable for average rusEFI with single bank

Comment (dron0gus) [reply firmware/tunerstudio/rusefi.input]: https://github.com/rusefi/rusefi/pull/4360

Comment (dron0gus) [reply firmware/tunerstudio/rusefi.input]: Another problem with big timeout (on linux):
TS waits on read and holds ttyACM0 device for 3 seconds. Meanwhile reconnection happens and device occupies next ttyACM1. Finally TS releases ttyACM0 and tries to reopen it. Bit it does not exits any more.

**Comment by dron0gus** [comment]
@rusefillc some explanation added.

**Comment by mck1117** [comment]
@rusefillc I guess we just ignore my PR comments now ☹️

**Comment by rusefillc** [comment]
@mck1117 I have no idea what comment on this #4358 are you referring to.

**Comment by mck1117** [comment]
image

**Comment by rusefillc** [comment]
image

is this github speak for "I have so many things to say but so far those comments are only visible to myself?"

I believe that we have an RTFM moment @mck1117 here.

**Comment by mck1117** [comment]
wow that's a really stupid "feature", it is indeed an RTFM

**Pull request merged**